docs/user-docs/latest/users-roles-permissions/configuring-administrator-roles.md

```diff
@@ -0,0 +1,164 @@
+# Configuring administrator roles
+
+Administrators are the users of an admin panel of a Strapi application. Administrator accounts and roles are managed with the Role-Based Access Control (RBAC) feature. It is available in the *Administration panel* section of the Settings interface, accessible from *General > Settings* in the main navigation of the admin panel.
+
+The *Administration panel* section is divided into 2 sub-sections: *Roles* and *Users* (see [Managing administrators](managing-administrators.md)).
+
+![Administrator roles interface](../assets/users-permissions/administrator_roles.png)
+
+The *Roles* sub-section of *Administration panel* displays all created roles for the administrators of your Strapi application.
+
+From this interface, it is possible to:
+
+- create a new administrator role (see [Creating a new role](#creating-a-new-role)),
+- delete an administrator role (see [Deleting a role](#deleting-a-role)),
+- or access information regarding an administrator role, and edit it (see [Editing a role](#editing-a-role)).
+
+By default, 3 administrator roles are defined for any Strapi application:
+
+- Author: to be able to create and manage their own content.
+- Editor: to be able to create content, and manage and publish any content.
+- Super Admin: to be able to access all features and settings. This is the role attributed by default to the first administrator at the creation of the Strapi application.
+
+::: warning ATTENTION
+If you use your Strapi application with the Community Edition (see [Pricing and Plans](https://strapi.io/pricing)), your use of the RBAC feature will be limited. Only the 3 default roles are indeed available, as you cannot create more roles and cannot delete the default ones. It is however possible to edit them, but to an extent:
+
+- You can only configure permissions for the content types, but not for the plugins and settings of the Strapi application.
+- Configuring permissions in detail is only available for the Enterprise Edition. With the Community Edition, although you can choose which fields of a content type are accessible, these fields are automatically fully accessible with all permissions.
+- Custom conditions defined for a specific permission are also only available for the Enterprise Edition.
```

Comment:
Maybe add a TO-DO here to link to the conditions document @pwizla is working on

Reply:
Actually we already talked about it with @pwizla and we plan on linking the conditions dev doc in Configuring administrator roles > Setting custom conditions for permissions. We already wrote the note, only the link is missing 🙂